oat-sa/extension-tao-item
# Breaking/nex 91/integrate npm item runner

Jira ticket: https://oat-sa.atlassian.net/browse/NEX-91
Vendorisation of item runner (reintegration)

**What I did:**
- deleted all js files & test files which were moved to `tao-item-runner-fe`
- registered new npm paths in php via install script & updater
- created `package.json` to install the new npm package
- enabled composer-npm-bridge to automatically install packages

**Prerequisites:**
- [x] https://github.com/oat-sa/tao-item-runner-fe/pull/6
- [x] release & publish on npmjs `tao-item-runner` @ 0.3.0
- [x] update above reference in this PR's package.json, & commit package-lock

**How to test:**
- Install this version of the `taoItems` extension in your TAO platform.
- Verify that the `@oat-sa/tao-item-runner` npm package was installed to `views/node_modules`.
- Verify that TAO backoffice and test delivery still work with no 404 errors (DEBUG mode).
- Verify that `npx grunt taoitemsbundle` bundle generation task completes, and site works (PROD mode)
- Verify that `npx grunt connect:dev taoitemstest` finds and runs the remaining tests
